modules/firewall/variables.tf

```diff
@@ -1,32 +1,26 @@
-variable "install_id" {
-  type        = string
-  description = "Identifier for install to apply to resources"
+variable "health_check_ip_cidr_ranges" {
+  default     = ["35.191.0.0/16", "209.85.152.0/22", "209.85.204.0/22", "130.211.0.0/22"]
+  description = "The list of GCP health check IP address ranges from which health check traffic will be authorized to flow, expressed in CIDR notation."
```

Comment:
does it make sense to add to this description about where the defaults were pulled from in the case of an update from gcp on their cidr ranges before we update?

Reply:
Fixed by https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/58/commits/f11facc71398d210c20bc68fd64a3a26831a91bd.